incubator/group/types.proto

```diff
@@ -139,9 +139,6 @@ message MsgProposeBase {
     // Proposers signatures will be counted as yes votes
     repeated bytes proposers = 2 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
     string comment = 3;
-    // ExecNow will attempt to execute the proposal immediately based on
```

Comment:
Don't we need this anymore? How we gonna handle this scenario mentioned in #53 

> _As a group member I want to submit a proposal that is executed **immediately** when a **ExecNow** value is set and a sufficient number of group members have signed the message in order to ...?_

Reply:
#53 is not part of #36 (anymore). I had separated the scenario into a new issue so that it can be addresses by a separate PR later. We wanted to focus on the critical path first and #36 is a special scenario with a very limited use case. I had commented about the constraints in the issue.
A workaround for it is to use the normal propose, vote, execute.